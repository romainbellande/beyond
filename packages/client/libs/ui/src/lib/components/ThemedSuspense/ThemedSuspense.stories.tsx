import { Story, Meta } from '@storybook/react';
import { ThemedSuspense } from './ThemedSuspense';

export default {
  component: ThemedSuspense,
  title: 'ThemedSuspense',
} as Meta;

const Template: Story = (args) => <ThemedSuspense {...args} />;

export const Primary = Template.bind({});
Primary.args = {};
