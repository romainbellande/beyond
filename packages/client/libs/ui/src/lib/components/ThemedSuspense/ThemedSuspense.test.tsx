import { render } from '@testing-library/react';
import { ThemedSuspense } from './ThemedSuspense';
import '@testing-library/jest-dom';

test('it can render', async () => {
  const { baseElement } = render(<ThemedSuspense />);

  expect(baseElement).toMatchSnapshot();
});
