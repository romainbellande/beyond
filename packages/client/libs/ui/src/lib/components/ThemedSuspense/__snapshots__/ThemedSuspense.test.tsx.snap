// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it can render 1`] = `
<body>
  <div />
</body>
`;
